engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
+
+2. Should user be able to upgrade or downgrade subscriptions?
+
+3. Should confirmations emails be allowed?
```

Comment:
Thinking about it again... anyone who can make braintree send emails on the providers behalve can probably also send fake emails themselves.